string obj_id
int32 obj_num
time time
float64[] boundingBox
int32 plastictype
int32 state